torch/csrc/jit/codegen/cuda/codegen.cpp

```diff
@@ -582,34 +582,42 @@ class CudaKernelGenerator : private OptInConstDispatch {
     TORCH_INTERNAL_ASSERT(tv->domain()->nDims() > 0);
     TORCH_INTERNAL_ASSERT(node->size() != nullptr);
 
-    switch (tv->memoryType()) {
-      case MemoryType::Global:
-        indent() << "// Allocate global tensor " << gen(tv) << "\n";
-        break;
-      case MemoryType::Shared:
-        if (node->size()->isConstScalar()) {
-          // Static shared memory
-          indent() << "__shared__ " << node->buffer_type() << " " << gen(tv)
-                   << "[" << genInline(node->size()) << "];\n";
-        } else {
-          // Align Offset Position
-          indent() << "offset = alignBufferSize(offset,"
-                   << dataTypeSize(node->buffer_type()) << ");\n";
-          // Shared Memory Pointer
-          indent() << node->buffer_type() << "* " << gen(tv)
-                   << " = reinterpret_cast<" << node->buffer_type() << "*>"
-                   << "(array + offset);\n";
-          // Increment Offset Position
-          indent() << "offset += (" << genInline(node->size()) << " * sizeof("
-                   << node->buffer_type() << "));\n";
-        }
-        break;
-      case MemoryType::Local:
-        indent() << node->buffer_type() << " " << gen(tv) << "["
-                 << genInline(node->size()) << "];\n";
-        break;
-      default:
-        TORCH_INTERNAL_ASSERT(false, "Unexpected memory type");
+    if (node->alias() != nullptr) {
+      // Allocate alias another Allocate node
+      const auto alias_tv = node->alias()->buffer()->as<kir::TensorView>();
+      indent() << node->buffer_type() << "* " << gen(tv) << " = "
+               << gen(alias_tv) << "; // Alias Allocate\n";
```

Comment:
Is that end of line comment a left over?

In general I like the comments outlining the code generation, but please move them "on top" (separate lines before the actual code).